projects/components/src/lib/directives/show-clipped-text.directive.spec.ts

```diff
@@ -69,67 +65,61 @@ xdescribe('ShowClippedTextDirective', () => {
+            tick(helper.hideDelay - 1);
             helper.moveMouseOverTooltip();
             expect(helper.isTooltipVisible).toBe(true);
-        });
+        }));
 
-        it('hides the tooltip if the mouse moves away from the tooltip', async function(this: Test): Promise<void> {
+        it('hides the tooltip if the mouse moves away from the tooltip', fakeAsync(function(this: Test): void {
             const helper = this.clippedTextHelper;
             helper.disabled = false;
             helper.componentInstance.directive.mouseoutDelay = 2; // TODO: make setter
             helper.width = '10px';
             helper.moveMouseOverHost();
             helper.moveMouseOffHost();
-            await timeout(1);
+            tick(1);
             helper.moveMouseOverTooltip();
             helper.moveMouseOffTooltip();
-            await timeout(10);
+            tick(10);
             expect(helper.isTooltipVisible).toBe(false);
-        });
+        }));
 
-        it('does not hide tooltip if the mouse quickly returns to host element', async function(this: Test): Promise<
-            void
-        > {
+        it('does not hide tooltip if the mouse quickly returns to host element', fakeAsync(function(this: Test): void {
             const helper = this.clippedTextHelper;
             helper.width = '10px';
             helper.moveMouseOverHost();
             helper.moveMouseOffHost();
-            await timeout(1);
+            tick(1);
             helper.moveMouseOverHost();
             expect(helper.isTooltipVisible).toBe(true);
-        });
+        }));
 
-        // TODO https://jira.eng.vmware.com/browse/VDUCC-116
-        // it('changes visibility to hidden after the tooltip fades out', async function(this: Test): Promise<void> {
-        //     const helper = this.clippedTextHelper;
-        //     helper.width = '10px';
-        //     helper.moveMouseOverHost();
-        //     // To avoid waiting a whole second
-        //     helper.componentInstance.directive.mouseoutDelay = 1;
-        //     helper.transitionTime = '1ms';
-        //     helper.moveMouseOffHost();
-        //     // Required for the transition to end, trying to set it to the 1ms above does not work, 25ms works sometimes
-        //     // 50 seems to work all the time
-        //     // Potential for flakiness here ¯\_(ツ)_/¯
-        //     await timeout(50);
-        //     expect(helper.tooltipVisibility).toBe('hidden', 'CSS visibility should have been set to hidden');
-        // });
+        it('changes visibility to hidden after the tooltip fades out', fakeAsync(function(this: Test): void {
+            const helper = this.clippedTextHelper;
+            helper.width = '10px';
+            helper.moveMouseOverHost();
+            // To avoid waiting a whole second
+            helper.componentInstance.directive.mouseoutDelay = 1;
+            helper.transitionTime = '1ms';
+            helper.moveMouseOffHost();
+            tick(50);
```

Comment:
If we are calling the method ourselves, we shouldn't need the tick for 50 seconds, right? Also shouldn't need changing the transition time, right? I think `helper.transitionTime` is also not needed anywhere else so it could be deleted